scripts/setIsLastCommentAdmin.js

```diff
@@ -0,0 +1,35 @@
+const admin = require('firebase-admin')
+const { initFirebaseAdmin } = require('./utils/initFirebaseAdmin.js')
+
+initFirebaseAdmin()
+
+const setIsLastCommentAdmin = async () => {
+  try {
+    const requests = admin.firestore().collectionGroup('requests')
+
+    const snapshots = await requests.get()
+
+    await Promise.all(
+      snapshots.docs.map(async (doc) => {
+        const responses = await admin
+          .firestore()
+          .collection(`users/${doc.ref.parent.parent.id}/requests/${doc.id}/responses`)
+          .orderBy('createdAt', 'desc')
+          .limit(1)
+          .get()
+
+        responses.docs.map((response) => {
+          doc.ref.update({
+            isLastCommentAdmin: response.data().hasOwnProperty('answeredBy')
+          })
+        })
```

Comment:
Don't use `map` if you're only iterating over an Array. It's meant to **create a new array with a mapping of the values of the previous one**. Use `forEach` if you only need to iterate.

**However**, is it absolutely necessary to iterate here? The query will return only one document because of the `limit(1)`.

**Also**, you should `await` the `update` operation.